client/me/privacy/main.jsx

```diff
@@ -158,7 +158,7 @@ const Privacy = createReactClass( {
 						>
 							{ this.state.submittingForm
 								? translate( 'Saving…' )
-								: translate( 'Save Privacy Settings' ) }
+								: translate( 'Save privacy settings' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.